composer.json

```diff
@@ -16,18 +16,18 @@
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "symfony/yaml": "~2.3||~3.0||~4.0||~5.0",
```

Comment:
why do you drop 5.0 support? that was the main reason for the previous compatibility pr

Reply:
This PR is not built on that PR

Reply:
In the master we already support SF5. Why do you drop it?

Reply:
I tried to restore these ~5.0 versions and it looks like a deeper level of investigation will be needed.
![Screenshot from 2020-06-15 10-17-13](https://user-images.githubusercontent.com/379322/84633983-774b5b00-aef1-11ea-926e-1fe5f16de745.png)

Reply:
I attempted this again and we are not able to run as-is with 5.x, I managed to upgrade a few deps but a lot will be stuck in 4.x until we have a fix.